lldb/include/lldb/Target/Target.h

```diff
@@ -1615,7 +1615,7 @@ class Target : public std::enable_shared_from_this<Target>,
 
   TargetStats &GetStatistics() { return m_stats; }
 
-private:
+protected:
```

Comment:
If you're wondering, this is so the unit test can access the private constructor.